.ci/generate_bear_requirements.py

```diff
@@ -214,6 +224,8 @@ def sort_requirements(req_dict):
     requirements['gem_requirements'] = get_gem_requirements(gem_reqs)
     requirements['npm_requirements'] = get_npm_requirements(npm_reqs)
     requirements['pip_requirements'] = get_pip_requirements(pip_reqs)
+    requirements['r_requirements'] = get_r_requirements(r_reqs)
```

Comment:
@jayvdb Maybe it is better if we keep these alphabetically sorted. The diff will be large and will need careful reviewing but, probably better in the long term.

Reply:
@anctartica has a fix for that at https://github.com/coala/coala-bears/pull/2625
We can merge that first, then this one